README.md

```diff
@@ -3,6 +3,7 @@ An Open Source Socialmedia Platform and Public forum for Questions and Discussio
 
 ## 🔗 Links
 - **UI Kit (Docs/Preview) 👉 [Here](http://mumble-lp.s3-website-us-west-2.amazonaws.com/)**
+- Discord [Join here](https://discord.com/invite/Tfr67ehuJH)
```

Comment:
Let's make it bold like the line above...
```suggestion
- Discord [Join here](https://discord.com/invite/Tfr67ehuJH)
+ **Discord [Join here](https://discord.com/invite/Tfr67ehuJH)**
```

Reply:
Thank you I will make the change